packages/widgets/src/indexer-manager/component.tsx

```diff
@@ -0,0 +1,72 @@
+import { useMemo } from "react";
+import { Anchor, Button, Card, Flex, Group, ScrollArea, Text } from "@mantine/core";
+import { useDisclosure } from "@mantine/hooks";
+import { IconCircleCheck, IconCircleX, IconReportSearch, IconTestPipe } from "@tabler/icons-react";
+
+import { clientApi } from "@homarr/api/client";
+import { useI18n } from "@homarr/translation/client";
+
+import type { WidgetComponentProps } from "../definition";
+import { NoIntegrationSelectedError } from "../errors";
+
+export default function IndexerManagerWidget({
+  options,
+  integrationIds,
+  serverData,
+}: WidgetComponentProps<"indexerManager">) {
+  if (integrationIds.length === 0) {
+    throw new NoIntegrationSelectedError();
+  }
+  const t = useI18n();
+  const data = useMemo(() => (serverData?.initialData ?? []).flatMap((indexers) => indexers.indexers), [serverData]);
+
+  const [loading, { toggle }] = useDisclosure();
```

Comment:
Instead you can use `isPending` from the `useMutation` hook below. It reflects the loading state